CI/qt-silent-install.qs

```diff
@@ -27,7 +27,7 @@ Controller.prototype.ComponentSelectionPageCallback = function() {
     var widget = gui.currentPageWidget();
 
     widget.deselectAll();
-    widget.selectComponent("qt.qt5.51210.win32_mingw73");
+    widget.selectComponent("qt.qt5.5152.win32_mingw73");
     widget.selectComponent("qt.tools.win32_mingw730");
```

Comment:
Likewise:
```suggestion
    widget.selectComponent("qt.tools.win32_mingw810");
```

OTOH Do we *need* to installing things that are already potentially already on AppVeyor?

Reply:
We don't, it should autodetect and skip install as it does now. This is only for the desktop installation